ui/pages/SingleAsset.tsx

```diff
@@ -98,14 +98,23 @@ export default function SingleAsset(): ReactElement {
       localizedDecimalAmount: undefined,
     }
 
-  const isUntrusted = isUntrustedAsset(asset)
-  const isUnverifiedByUser = isUnverifiedAssetByUser(asset)
+  const isTrusted = isTrustedAsset(asset)
+  const isVerified = isVerifiedAsset(asset)
   const [warnedAsset, setWarnedAsset] =
     useState<SmartContractFungibleAsset | null>(null)
 
   const showActionButtons = isEnabled(FeatureFlags.SUPPORT_UNVERIFIED_ASSET)
-    ? !isUnverifiedByUser
+    ? isTrusted
     : true
+  const showVerificationButton = isEnabled(
+    FeatureFlags.SUPPORT_UNVERIFIED_ASSET
+  )
+    ? !isVerified && !isTrusted
+    : false
+
+  const showVerifiedLabel = isEnabled(FeatureFlags.SUPPORT_UNVERIFIED_ASSET)
+    ? isVerified
+    : false
```

Comment:
```suggestion
  const showVerifiedLabel =
    isEnabled(FeatureFlags.SUPPORT_UNVERIFIED_ASSET) && isVerified
```

Would also like to suggest inlining these checks if they're not too complex. Non-blocking

Reply:
Agree, once the flag is dropped we can just drop `showVerifiedLabel` and use `isVerified` instead.

Reply:
https://github.com/tahowallet/extension/pull/3528/commits/22ee963102b1f2248c53f0160fda380c7d4db899